modules/wallet/consts.go

```diff
@@ -15,6 +15,15 @@ const (
 	// defragThreshold is the number of outputs a wallet is allowed before it is
 	// defragmented.
 	defragThreshold = 50
+
+	// rebroadcastInterval is the number of blocks the wallet will wait until
+	// it rebroadcasts an unconfirmed transaction by adding it to the
+	// transaction pool again.
+	rebroadcastInterval = 10
+
+	// rebroadcastMaxTries is the maximum number of times a transaction set
+	// will be rebroadcasted before the wallet stops tracking it
+	rebroadcastMaxTries = 10
```

Comment:
There's a const called `RespendTimeout` in `wallet.go` (that should be moved to `consts.go`) which says the wallet will try to spend an output again after 40 blocks of it not being spent. Instead of having `rebroadcastMaxTries`, you should just stop trying once the respend timeout has expired.

Now that we are trying to rebroadcast transactions, we can probably jump the `RespendTimeout` to 72 blocks, which is 12 hours.